src/strategies/erc3525-vesting-voucher/index.ts

```diff
@@ -8,95 +8,90 @@ export const author = 'buchaoqun';
 export const version = '0.1.1';
 
 const abi = [
-  'function getSnapshot(uint256 tokenId_) view returns (uint8 claimType_, uint64 term_, uint256 vestingAmount_, uint256 principal_, uint64[] maturities_, uint32[] percentages_, uint256 availableWithdrawAmount_, string originalInvestor_, bool isValid_)',
-  'function balanceOf(address owner) view returns (uint256)',
-  'function tokenOfOwnerByIndex(address owner,uint256 index) view returns (uint256)'
+    'function getSnapshot(uint256 tokenId_) view returns (uint8 claimType_, uint64 term_, uint256 vestingAmount_, uint256 principal_, uint64[] maturities_, uint32[] percentages_, uint256 availableWithdrawAmount_, string originalInvestor_, bool isValid_)',
+    'function balanceOf(address owner) view returns (uint256)',
+    'function tokenOfOwnerByIndex(address owner,uint256 index) view returns (uint256)'
 ];
 
 export async function strategy(
-  space,
-  network,
-  provider,
-  addresses,
-  options,
-  snapshot
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
 ) {
```

Comment:
@buchaoqun make sure it is not changing other strategies, revert changes from these files plz :)